sw/README.md

```diff
@@ -37,64 +35,7 @@ Install in `--prefix`:
 make install
 ```
 
-# Building Software
-The sw directory is divided in two sub-directories:
-* host - software for the host
-* SoC  - software for the SoC
-
-### Directory structure
-```
-.
-├── host                                  # Host software
-│   └── <project_name>                    # Project directory
-│       ├── Makefile                      # Makefile
-│       └── src                           # Sources
-│           └── ...
-├── Makefile                              # Top level Makefile
-└── SoC                                   # SoC software
-    ├── examples                          # Example projects
-    │   └── <project_name>                # Project directory
-    │       ├── Makefile                  # Makefile
-    │       └── src                       # Sources
-    │           └── ...
-    └── template                          # Template project
-    |   ├── Makefile
-    |   └── src
-    |       └── main.c
-    ├── linker                            # Linker script directory
-    │   └── <LINKER_SCRIPT>.ld
-    ├── startup                           # Startup directory
-    │   └── startup.s
-
-
-├── host                                  # Host software
-│   └── <project_name>                    # Project directory
-│       ├── Makefile                      # Makefile
-│       └── src                           # Sources
-│           └── ...
-├── Makefile                              # Top level Makefile
-└── SoC                                   # SoC software
-    ├── examples                          # Example projects
-    │   └── <project_name>                # Project directory
-    │       ├── Makefile                  # Makefile
-    │       └── src                       # Sources
-    │           └── ...
-    ├── linker                            # Linker script directory
-    │   └── <LINKER_SCRIPT>.ld
-    ├── startup                           # Startup directory
-    │   └── startup.s
-
-```
-
-The top level Makefile contains targets for both host software and SoC software.
-
-### Build command
-To build software for the host:
-```
-make host_<project_name>
-```
-To build software for the SoC:
-```
-make soc_<project_name>
-```
-The relative output product (objs, `.bin`, `.elf` ...) can be found in the `sw/[host|SoC]/<project_name>` directory.
+# Software Tree Structure
+The sw directory is organized in two major components:
+* host - Contains software that runs on the host side, typically x86-based systems. This includes Linux drivers for interfacing with the UninaSoC.

```

Comment:
this is only true for the HPC config, maybe it should be clarified

Reply:
I agree. Solved.